packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -1994,6 +1994,156 @@ class FavoritesTable extends React.Component {
+    - `posinset` - number representing where in the order this node sits amongst its siblings 
+    - `setsize` - number representing the number of children this node has
+    - `isChecked` - (optional) if this row uses checkboxes, flag indicating the checkbox checked
+3. Use the `treeRow` cellTransform in the first column of the table. `treeRow` expects one or two callbacks as params.
+    - `onCollapse` - Callback when user expands/collapses a row to reveal/hide the row's children.
+    - `onCheckChange` - (optional) Callback when user changes the checkbox on a row.
+
+```js
+import React from 'react';
+import { Table, TableHeader, TableBody, headerCol, treeRow } from '@patternfly/react-table';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+
+class TreeTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: [
+        {
+          repositories: 'Repositories one',
+          branches: 'Branch one',
+          pullRequests: 'Pull request one',
+          workspaces: 'Workplace one',
+          children: [
+            {
+              repositories: 'Repositories two',
+              branches: 'Branch two',
+              pullRequests: 'Pull request two',
+              workspaces: 'Workplace two',
+              children: [
+                {
+                  repositories: 'Repositories three',
+                  branches: 'Branch three',
+                  pullRequests: 'Pull request three',
+                  workspaces: 'Workplace three',
+                }
+              ]
+            },
+            {
+              repositories: 'Repositories four',
+              branches: 'Branch four',
+              pullRequests: 'Pull request four',
+              workspaces: 'Workplace four',
+            },
+            {
+              repositories: 'Repositories five',
+              branches: 'Branch five',
+              pullRequests: 'Pull request five',
+              workspaces: 'Workplace five',
+            }
+          ]
+        },
+        {
+          repositories: 'Repositories six',
+          branches: 'Branch six',
+          pullRequests: 'Pull request six',
+          workspaces: 'Workplace six',
+          children: [
+            {
+              repositories: 'Repositories seven',
+              branches: 'Branch seven',
+              pullRequests: 'Pull request seven',
+              workspaces: 'Workplace seven'
+            }
+          ]
+        },
+        {
+          repositories: 'Repositories eight',
+          branches: 'Branch eight',
+          pullRequests: 'Pull request eight',
+          workspaces: 'Workplace eight'
+        }
+      ],
+      expandedRows: ['Repositories one', 'Repositories six'],
+      checkedRows: []
+    };
+    
+    this.buildRows = ([x, ...xs], level, posinset, isHidden = false) => {
```

Comment:
can you add a comment block to this function explaining what the params are and what is being returned?